src/vnm_mad/remotes/lib/vm.rb

```diff
@@ -65,6 +53,19 @@ def each_nic(block)
                 end
             end
 
+            # Get NIC_ALIAS by NIC_ID
+            #   @param element [String] the NIC_ID
+            #   @return [Hash] the NIC_ALIAS
+            def nic_alias(id)
+                if @nic_aliases
```

Comment:
better use exit condition.

return if @nic_aliases.nil?

Reply:
I will change it